power management : build failure on stm32u083_dk board

### Describe the bug

There is a build failure on STM32U0 boards for power management samples.

This issue was introduced with the addition of PM support on STM32U0 boards through this PR: https://github.com/zephyrproject-rtos/zephyr/pull/88963.

The build error suggests using devicetree instead of the deprecated Kconfig option.

### Regression

- [ ] This is a regression.

### Steps to reproduce

1. Go to zephyr workspace
2. run with west: 
`west build -p -b stm32u083c_dk/stm32u083xx samples/boards/st/power_mgmt/blinky -T sample.boards.stm32.power_mgmt.blinky`
2. or run with twister :
` ./scripts/twister -p stm32u083c_dk  -T samples/boards/st/power_mgmt/blinky`/ 
3. see log output 

### Relevant log output

```shell
/zephyrproject/zephyr/drivers/timer/stm32_lptim_timer.c:34:2: error: #warning Kconfig for LPTIM source clock (LSI/LSE) is deprecated, use device tree. [-Werror=cpp]
   34 | #warning Kconfig for LPTIM source clock (LSI/LSE) is deprecated, use device tree.
      |  ^~~~~~~
```

### Impact

Annoyance – Minor irritation; no significant impact on usability or functionality.

### Environment

- OS: Linux ubuntu

- Toolchain: Zephyr SDK 0.17.O

- Zephyr version: v4.1.99

### Additional Context

_No response_

--- a/boards/st/stm32u083c_dk/stm32u083c_dk.dts
+++ b/boards/st/stm32u083c_dk/stm32u083c_dk.dts
@@ -134,6 +134,12 @@
 	};
 };
 
+stm32_lp_tick_source: &lptim2 {
+	clocks = <&rcc STM32_CLOCK(APB1, 30)>,
+		<&rcc STM32_SRC_LSI LPTIM2_SEL(1)>;
+	status = "okay";
+};
+
 &tsc {
 	status = "okay";
 	pinctrl-0 = <&tsc_g1_io1_pb12 &tsc_g1_io2_pb13 &tsc_g6_io1_pd10 &tsc_g6_io2_pd11>;

--- a/dts/arm/st/u0/stm32u0.dtsi
+++ b/dts/arm/st/u0/stm32u0.dtsi
@@ -573,6 +573,7 @@
 			reg = <0x40009400 0x400>;
 			interrupts = <18 1>;
 			interrupt-names = "combined";
+			status = "disabled";
 		};
 
 		tsc: tsc@40024000 {

